constraint/pkg/client/client_test.go

```diff
@@ -294,6 +294,15 @@ func TestClient_AddTemplate(t *testing.T) {
 			wantHandled: map[string]bool{handlertest.TargetName: true},
 			wantError:   nil,
 		},
+		{
+			name:    "Long name",
```

Comment:
You are adding a test here, where are you adding the validation?
> If at most 63 characters, then ConstraintTemplateNames can be up to 253-63=190 characters long.

Reply:
The validation is already happening on K8's side, since CRD kinds have a maximum size limit. This is just double checking that the check remains in place.

Reply:
i.e. the apiextension validation code we're calling performs this check

Reply:
I thought Constraints have a maximum name length of 63 chars, not constraint template. This test is throwing an error if the template name is greater than 63 chars?

Reply:
Will's comment [in the issue](https://github.com/open-policy-agent/frameworks/issues/116#issuecomment-1088829287) confused me
> If at most 63 characters, then ConstraintTemplateNames can be up to 253-63=190 characters long.

Reply:
yeah, I don't think we knew the validation code was being activated regardless